docs/uppy-core.mdx

```diff
@@ -345,6 +346,16 @@ purposes via [`uppy.log()`](#logmessage-type).
 <details>
   <summary>Filter, change, and abort example</summary>
 
+Allow all files, also duplicate files. This will override the file if it has not
```

Comment:
```suggestion
Allow all files, also duplicate files. This will overwrite the file (clear meta data, reset image editor modifications) if it has not
```

Reply:
How about this? because replace means _everything_ you could possibly do with uppy files.

```suggestion
Allow all files, also duplicate files. This will replace the file if it has not
```